graphs/graph_modules/top_10_users_graph.py

```
@@ -0,0 +1,218 @@
+            # Process validated series data
+            for serie in series:
+                if serie["name"] == "TV":
+                    processed_data["tv_data"] = serie["data"]
+                elif serie["name"] == "Movies":
+                    processed_data["movie_data"] = serie["data"]
+
+            return processed_data
+
+        except Exception as e:
+            logging.error(f"Error processing top 10 users data: {e}")
+            raise DataValidationError(f"Failed to process data: {str(e)}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)